modules/ossm-rn-new-features.adoc

```diff
@@ -15,11 +15,33 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.3.2
+
+This release of {SMProductName} introduces new features, addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.9 or later.
```

Comment:
I don't think we introduced any new features. It's only bug and CVE fixes. But I'd like others to confirm.